File: 100-Days-Series/Day 19/RemoveDuplicatesElements.c
#include<stdio.h>
void getRemoveDuplicatesElements(int arr[],int *size){
    int temp[*size];
    int i,j,index=0;
    for(i=0;i<*size;i++){
        for(j=0;j<index;j++){
            if(arr[i]==temp[j]){
                break;
            }
        }
        if(j==index)
            temp[index++]=arr[i];
    }
    // Copying unique elements back to the original array
    for(i=0;i<index;i++){
        arr[i]=temp[i];
    }
    *size=index;// Updating size to reflect unique elements
}
// Let's Start here 
void main(){
    int n,i;
    printf("Enter the number of elements in the array: ");
    scanf("%d", &n);
    int arr[n]; // An Array Created
    printf("Enter the elements of the array:\n");
    for (i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }
    // Call the function for remove duplicate elements
    getRemoveDuplicatesElements(arr, &n);
    // Print after removing duplicates elements
    printf("Array after removing duplicates:\n");
    for (i = 0; i < n; i++) {
        printf("%d ", arr[i]);
    }

}

File: 100-Days-Series/Day 19/RemoveDuplicatesElements.js

// Arrow Function for remove duplicate elements of an array
let size;
const getRemoveDuplicateElements=(arr)=>{
    size=arr.length;
    temp=new Array();
    let index=0;
    let j;
    for(let i=0;i<size;i++){
        for(j=0;j<index;j++){
            if(arr[i]==arr[j])
                break;
        }
        if(j==index){
            temp[index++]=arr[i];
        }
    }
    size=index;
    return temp;
}

// Let's Start here
// An Array Constant values.
arr=new Array(1,2,3,1,1,3,4,5);
// Call the function for remove duplicate elements.
carr=getRemoveDuplicateElements(arr);
// Array after removing duplicates elements 
for(let i=0;i<size;i++){
    console.log(" "+carr[i]);
}